decred_wallet/view_controller/WalletLogViewController.swift

```diff
@@ -7,13 +7,17 @@
 // license that can be found in the LICENSE file.
 
 import UIKit
+import JGProgressHUD
 
 class WalletLogViewController: UIViewController {
     @IBOutlet weak var logTextView: UITextView!
-    
+    static var progressHud = JGProgressHUD(style: .light)
```

Comment:
- Don't make this static please. Dismiss the hud in `viewDidLoad`, after reading the log.
- Declare but don't assign here. Assign in `viewDidLoad`